### PR TITLE
default colors for enum ColorMap type

### PR DESCRIPTION
**Pull request by scseeman**

### CODE REVIEW

**Comment by j9ac9k** [comment]
CI failed on a test that intermittently glitches out... re-running the failed pipeline.

File: pyqtgraph/widgets/ColorMapWidget.py [hunk 1]
```diff
@@ -71,7 +71,15 @@ def addNew(self, name):
         defaults = fieldSpec.get('defaults', {})
         for k, v in defaults.items():
             if k == 'colormap':
-                item.setValue(v)
+                if mode == 'range':
+                    item.setValue(v)
+                if mode == 'enum':
```
Comment (j9ac9k) [inline pyqtgraph/widgets/ColorMapWidget.py:76]: let's change this second `if` to be `elif`

**Comment by j9ac9k** [comment]
Hi @scseeman thanks for the PR.  I just had the one suggestion I added as an inline comment, and besides that, I'm good with everything else.

File: pyqtgraph/widgets/ColorMapWidget.py (hunk 1 above)
Comment (scseeman) [reply pyqtgraph/widgets/ColorMapWidget.py]: Done!

**Comment by j9ac9k** [comment]
Thanks for the PR @scseeman this looks good to me, merging!

**Pull request merged**